internal/servers/plugin/v3/plugin.go

```diff
@@ -438,10 +440,10 @@ func (s *Server) Transform(stream pb.Plugin_TransformServer) error {
 				}
 
 				if err := stream.Send(&pb.Transform_Response{Record: recordBytes}); err != nil {
-					return fmt.Errorf("error sending response: %w", err)
+					return status.Errorf(codes.Internal, "error sending response: %v", err)
 				}
-			case <-pluginStopsWriter:
-				return nil
+			case err := <-pluginStops:
+				return err
```

Comment:
Technically we won't need to return the error here since waiting on the error group will return the one from https://github.com/cloudquery/plugin-sdk/pull/2001/files#diff-e161c2b48e205f785f01f22f6f0157141c2b896e6805d4fba599b1f41c5a9fe0R421

Reply:
I removed this channel.